samples/templates/default-azuredeploy-docker-sql.json

```diff
@@ -145,6 +145,13 @@
                 "description": "Determines whether export will be enabled for this fhir instance. If true, a storage account will be created as part of the deployment. You will need owner or user-administrator permissions for this."
             }
         },
+        "enableDataConvert": {
+            "type": "bool",
+            "defaultValue": true,
+            "metadata": {
+                "description": "Determines whether data convert will be enabled for this fhir instance. If true, an Azure container registry will be created as part of the deployment. You will need owner or user-administrator permissions for this."
```

Comment:
I think "data convert" is not descriptive enough. Even "the $data-convert operation" would be clearer.

Reply:
Thanks. Updated